app/Makefile

```diff
@@ -19,5 +21,10 @@ dev-build-server:
 	cd ./server; \
 	npm run build:dev
 
+test-ui:
+	cd ./ui; \
```

Comment:
Same as above, sticking to what the rest of the file does. I feel like this needs a wider conversation about why makefile, what does it mean for npm scripts and how this whole system should work.